jsx_ppx/test/test.ml

```diff
@@ -0,0 +1,12 @@
+[@@@bs.config { jsx = 3 }]
+
+let div ~children  () = children
+
+module Test = struct
+
+let make ~a  ~b  _ =
+  print_endline (a ^ b);
+  print_endline "This function should be named `Test`";
+  ((div ~children:[] ())[@JSX ])[@@react.component ]
```

Comment:
Besides this file ideally being using Reason syntax, I wonder if using JSX format (`<div />`) would be possible too 🤔